api/internal/handler/resources/resources.go

```diff
@@ -77,7 +77,7 @@ func (h *Handler) ApplyRoute(r *gin.Engine, cfg config.Config) {
 	}
 
 	// add misc routes
-	r.GET("/apisix/admin/plugins/list", handler.Wrap(h.ProxyMisc, nil))
+	r.GET("/apisix/admin/plugins/*param", handler.Wrap(h.ProxyMisc, nil))
```

Comment:
`list` is enough. It is used to get the full table of plugins.

We have this usage, and you can verify it this way
- /apisix/admin/plugins/list
- /apisix/admin/schema/route
- /apisix/admin/schema/plugins/real-ip